### PR TITLE
Doc: updates

### PR DESCRIPTION
**Pull request by pbougue**
Also, small code improvement

### CODE REVIEW

File: documentation/slate/source/includes/apis.md [hunk 1]
```diff
@@ -260,35 +266,42 @@ paginate results.
...
-### Parameters
+### Shared parameters
 
 #### <a name="depth"></a>depth
 
-This tiny parameter can expand Navitia power by making it more wordy.
+You are looking for something, but Navitia doesn't output it in your favorite endpoint?
+You want to request more from navitia feed?
+You are receiving feeds that are too important and too slow with low bandwidth?
+You would like it to be GraphQL?
```
Comment (stifoon) [inline documentation/slate/source/includes/apis.md]: Talking about GraphQL should be an issue. If you write about it in the doc, and navitia does not manage it, it should be deceptive, no?

File: documentation/slate/source/includes/apis.md [hunk 2]
```diff
@@ -1208,8 +1230,36 @@ You just have to verify that the coordinates of the geocoded object are inside t
...
+So if one doesn't understand why a place is inside or outside an isochrone,
+please compute a journey from the "center" of isochrone to that precise place.
+
+To do that, just 3 changes are needed:
+
+- provide a starting `datetime` to compare arrival time evenly
+- change endpoint: `/isochrones` to `/journeys`
+- provide a destination using `&to=<my_place>`
+
+Please remember that isochrones use crowfly at the end so they are less precise than journeys.
```
Comment (stifoon) [inline documentation/slate/source/includes/apis.md]: "less precise than journeys" and heat_maps (with heat_maps link :) )

File: documentation/slate/source/includes/isochrones.md [hunk 3]
```diff
@@ -5,15 +5,15 @@
...
+your reach at a given time and their corresponding travel times, using a variety of transportation options.
 You can even specify the maximum amount of time you want to spare on travel and find
 the quickest way to reach your destination.
 
 [![a simple isochrone request on Navitia playground](isochrones_example.png)](http://canaltp.github.io/navitia-playground/play.html?request=https%3A%2F%2Fapi.navitia.io%2Fv1%2Fcoverage%2Fsandbox%2Fisochrones%3Ffrom%3D2.377097%3B48.846905%26max_duration%3D2000%26min_duration%3D1000&token=3b036afe-0110-4202-b9ed-99718476c2e0)
 
 Isochrone computing exposes information under two formats:
 
--   either [Journeys](#journeys) service which provides a list with all the reachable stops from a potential destination
+-   either [Journeys](#journeys) service which provides a list with all the reachable stops at a moment from a potential destination
```
Comment (stifoon) [inline documentation/slate/source/includes/isochrones.md]: at a time ?

File: documentation/slate/source/includes/apis.md [hunk 4]
```diff
@@ -1169,12 +1186,12 @@ Also known as `/isochrones` service.
...
 
 This service gives you a multi-polygon response which
-represent a same time travel zone: https://en.wikipedia.org/wiki/Isochrone_map
+represents a same time travel zone at a given moment: https://en.wikipedia.org/wiki/Isochrone_map
 
 As you can find isochrone tables using `/journeys`, this service is only another representation
 of the same data, map oriented.
 
-It is also really usefull to make filters on geocoded objects in order to find which ones are reachable within a specific time.
+It is also really usefull to make filters on geocoded objects in order to find which ones are reachable at a moment within a specific time.
```
Comment (stifoon) [inline documentation/slate/source/includes/apis.md]: at a time?

**Review by TeXitoi — APPROVED** [review]

**Review by kadhikari — APPROVED** [review]

**Pull request merged**